akka-persistence/src/main/resources/reference.conf

```diff
@@ -71,6 +72,7 @@ akka.persistence {
                 parallelism-max = 8
             }
         }
+        # Default dispatcher for streaming snapshot IO
         default-stream-dispatcher {
```

Comment:
it's a bit weird, I'll move it into the file snapshot config, as that is the only place it is used